Song/SongWriter.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Song
+{
+    public class SongWriter
+    {
+        public string Sing(List<string> animals, List<string> animalLines)
+        {
+            VerseWriter verserWriter = new VerseWriter();
+
+            if (animals.Count == 1) 
+            {
+                return verserWriter.WriteVerse(animals, animalLines);
+            }
+            
+            else
+            {
+                List<string> newAnimals = new List<string>();
+                List<string> newAnimalsLines = new List<string>();
+                newAnimals.Add(animals[0]);
```

Comment:
`newAnimals` has no logical meaning. It's not new animals, it's animal you sung about - and thus should be named accordingly.